src/algebra/algebra/basic.lean

```diff
@@ -475,6 +475,55 @@ lemma algebra_map_End_eq_smul_id (a : R) :
   ((algebra_map K (End K V)) a).ker = ⊥ :=
 linear_map.ker_smul _ _ ha
 
+section
+
+variables {R M}
+
+lemma End_is_unit_apply_inv_apply {f : module.End R M} (h : is_unit f) (x : M) :
```

Comment:
Should the name contain `of_is_unit` since it is an assumption? Similarly below.

Reply:
done in [7b7ea3f](https://github.com/leanprover-community/mathlib/pull/15559/commits/7b7ea3f90eecd16f740a91b377b332cb672a105d)